<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataEntryWidget</class>
 <widget class="QWidget" name="DataEntryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>595</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>صفحة ادخال البيانات</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="3">
    <widget class="QLineEdit" name="productNameLineEdit"/>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="IDLineEdit"/>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="exitButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>الخروج</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="enterButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>ادخل البيانات</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string>الرقم المسلسل</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="providerNameLineEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>اسم المنتج</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="priceLineEdit"/>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>اسم المورد</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>السعر بالجنية</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
